code/datums/uplink_item.dm

```diff
@@ -477,12 +477,18 @@ var/list/uplink_items = list()
 	item = /obj/item/device/chameleon
 	cost = 6
 
-/datum/uplink_item/stealthy_tools/chameleon_proj
+/datum/uplink_item/stealthy_tools/smoke_bombs
```

Comment:
The path was the same as the chameleon projectors.
If the atomicity is *really* a problem I could remove it, but it was just a one-line fix.